fbw-a32nx/src/systems/instruments/src/MCDU/legacy_pages/A320_Neo_CDU_ProgressPage.ts

```diff
@@ -16,10 +16,14 @@ export class CDUProgressPage {
     };
     mcdu.activeSystem = 'FMGC';
     const flightNo = mcdu.flightNumber ?? '';
-    const flMax = mcdu.getMaxFlCorrected();
+    const recMaxFl = mcdu.getMaxFlCorrected();
+    const flMaxText = recMaxFl !== null ? `{magenta}FL:${recMaxFl.toString()}{end}` : `-----`;
```

Comment:
Is this colon intentional?
```suggestion
    const flMaxText = recMaxFl !== null ? `{magenta}FL${recMaxFl.toString()}{end}` : `-----`;
```

Reply:
oops, probably was a copilot suggestion. Should be fixed now alongside your other comments.